ui/src/app/shared/job-stream.ts

```diff
@@ -19,7 +19,7 @@ export class JobStream extends BehaviorSubject<JobListView> {
               private request: QueryJobsRequest) {
     super({
       results: [],
-      totalSize: 0,
+      totalSize: undefined,
```

Comment:
Not complaining (because I do think `undefined` *is* more appropriate in this case), just curious - would just changing this to `null` instead of `0` have solved the issue too?

Reply:
This line apparently doesn't really matter AFAICT. Just something I noticed while making the change. It just felt better to me that the "unloaded state" (which I think is never displayed), would show "of many" rather than "of 0".